spec/image_spec.rb

```diff
@@ -1,20 +1,20 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require 'fileutils'
 
 RSpec.describe Assembly::Image do
   let(:assembly_image) { described_class.new(input_path) }
   let(:input_path) { TEST_TIF_INPUT_FILE }
+  let(:output_file) { File.join(TEST_OUTPUT_DIR, File.basename(input_path).gsub('.tif', '.jp2')) }
 
```

Comment:
- we will use `input_path` and `output_file` in each spec instead of `TEST_TIF_INPUT_FILE` and `TEST_JP2_OUTPUT_FILE` since this lets us use unique input/output files for each spec
- `output_file` is now computer automatically based on the input_file (so can vary per spec)

Reply:
I think this is a great approach, but maybe it could be jp2_output_file, if that's what it always is?  Just for clarity.

Reply:
i changed the variable name to `jp2_output_file`